components/editor/emacs/Makefile

```diff
@@ -82,6 +82,9 @@ CONFIGURE_OPTIONS +=	--infodir=$(CONFIGURE_INFODIR)
 CONFIGURE_OPTIONS +=	--datarootdir=$(CONFIGURE_PREFIX)/share
 CONFIGURE_OPTIONS +=	--libexecdir=$(CONFIGURE_PREFIX)/lib
 CONFIGURE_OPTIONS +=	--with-gif=no
+# Oracle builds emacs without imagemagick and lcms2
```

Comment:
What exact option fixes mentioned issue?

Reply:
It was disabling imagemagick.

Reply:
imagemagick got disabled in 2017 (9760b3fce0) without further comment. lcms2 got disabled in 2019 when they updated to emacs-26.1 (6bb577666a) without further information.

Reply:
Then just disable imagemagick and add reference to redmine issue - it'll be more useful for the next person who touches this component.